backend/game/player/player.py

```diff
@@ -9,6 +9,7 @@ def __init__(self):
         self.is_alive = True
         self.victory_points = constants.DEATH_HIT_POINT
         self.energy = constants.DEFAULT_ENERGY_CUBE
+        self.card_on_hand = {}
```

Comment:
Discussed this in another review, cannot have dictionary store cards because can be duplicates, probably better to use a list

Reply:
yup, sorry. I will update my PR given player.py changes made to branch It_Has_A_Child

Reply:
you mentioned this was discussed, do you recall which PR?

Perhaps it was fixed already.

Reply:
Having a class for a hand seems silly and doesn't really address the issue if it's going to be a dictionary. It likely should be a list. The only issue may be that, if the player has two and wanted to discard the second one, the first may disappear which can be unsettling. We would then have to have something like a manual option which takes a card index.

This also gives me an interesting thought. Are there any cards that, if they have a special affect that happens, could cause another card to use its special affect? So like, if a player has a card they use, should it then go through their cards to ensure that there are no other special affects that should happen? It may also require that everyone's cards are checked for special affects if they can play off each other like that.